AccessCheckoutSDK/AccessCheckoutSDK/AccessClient.swift

```diff
@@ -111,8 +111,9 @@ extension AccessCheckoutClient: AccessClient {
                 completionHandler(.failure(error))
             }
         } else {
-            discovery.discover(urlSession: urlSession) {
-                if let url = self.discovery.verifiedTokensSessionEndpoint {
+            let vts = DiscoverLinks(service: "service:verifiedTokens", endpoint: "verifiedTokens:session")
```

Comment:
If you got an opportunity, feel free to move it into the DiscoverLinks class as a static constant so that it can also be used as part of the tests we'll write for the new Cvv only client class (and also so that all code related to DiscoverLinks stays togther, similarly to an enum)